zcash_client_sqlite/src/prepared.rs

```diff
@@ -516,19 +561,17 @@ impl<'a, P: consensus::Parameters> DataConnStmtCache<'a, P> {
     pub(crate) fn stmt_insert_address(
         &mut self,
         account: AccountId,
-        mut diversifier_index: DiversifierIndex,
+        diversifier_index: DiversifierIndex,
         address: &UnifiedAddress,
-    ) -> Result<i64, SqliteClientError> {
-        diversifier_index.0.reverse();
-        let sql_args: &[(&str, &dyn ToSql)] = &[
-            (":account", &u32::from(account)),
-            (":diversifier_index_be", &&diversifier_index.0[..]),
-            (":address", &address.encode(&self.wallet_db.params)),
-        ];
-
-        self.stmt_insert_address.execute(sql_args)?;
+    ) -> Result<(), SqliteClientError> {
+        self.stmt_insert_address.execute(
+            &self.wallet_db.params,
+            account,
+            diversifier_index,
```

Comment:
We are no longer reversing the bytes of `diversifier_index`, but the field is still called `diversifier_index_be`. Is that correct?

Reply:
We perform the reverse here: https://github.com/zcash/librustzcash/pull/672/files/f1f9465f375a0aa77e86b7f586f990d5e2d47a57#diff-1e7d5d95a00896e1916695c2509d888591b4eb647ec38b7e750855e0b72c609dR70